apps/app/components/account/email-code-form.tsx

```diff
@@ -66,18 +80,36 @@ export const EmailCodeForm = ({ onSuccess }: any) => {
       });
   };
 
+  useEffect(() => {
```

Comment:
Reason behind this UseEffect

Reply:
At line 83, the useEffect function is implemented to create a 30-second timer. This timer ensures that if the user requires a new login code, they must wait for 30 seconds before being able to request a new code.

Reply:
Wrap this logic inside a hook.